plutus-scb-client/src/View.purs

```diff
@@ -56,57 +56,56 @@ render (State { chainState, fullReport }) =
     ]
 
 fullReportPane :: forall p. Chain.State -> FullReport ContractExe -> HTML p HAction
-fullReportPane chainState fullReport@(FullReport { events, latestContractStatus, transactionMap, utxoIndex, annotatedBlockchain, walletMap }) =
+fullReportPane chainState fullReport@(FullReport { events, latestContractStatuses, transactionMap, utxoIndex, annotatedBlockchain, walletMap }) =
   row_
-    [ col10_ [ contractStatusPane latestContractStatus ]
-    , col12_ [ ChainAction <<< Just <$> annotatedBlockchainPane chainState walletMap annotatedBlockchain ]
+    [ col10_ [ contractStatusPane latestContractStatuses ]
+    , col12_ [ ChainAction <<< Just <$> annotatedBlockchainPane chainState (unwrap walletMap) annotatedBlockchain ]
     , col12_ [ eventsPane events ]
-    , col8_ [ transactionPane transactionMap ]
+    , col8_ [ transactionPane (unwrap transactionMap) ]
```

Comment:
would it make sense for `transactionPane` to take the wrapped up `transactionMap`?

Reply:
Yeah, agreed. 👍